storage/client_raft_test.go

```diff
@@ -68,7 +68,9 @@ func TestStoreRecoverFromEngine(t *testing.T) {
 
 	manual := hlc.NewManualClock(0)
 	clock := hlc.NewClock(manual.UnixNano)
-	eng := engine.NewInMem(proto.Attributes{}, 1<<20)
+	stopper := stop.NewStopper()
```

Comment:
Call the stoppers in this test and the next one `engineStopper` and `stopper` instead of `stopper` and `stopper2`.